Data Epic Project -1/main.py

```diff
@@ -0,0 +1,36 @@
+
+import gspread
+
+gc = gspread.service_account("gspread-sheet-2-4ebedcdae885.json")
+
+def open_spreadsheet(spread_sheet):

```

Comment:
Please include a docstring to describe your functions. The same thing applies to all other functions here.